README.md

```diff
@@ -7,7 +7,9 @@ Plain simple build step for setting up your project with [Carthage](https://gith
 You can cache the result of `carthage bootstrap` to make Bitrise faster when building unchanged dependencies. For that you'll have to [set up caching](http://devcenter.bitrise.io/caching/about-caching/) in your Bitrise workflow.
 
 1. add a cache pull step before your Carthage step,
-2. set up a cache push step after your Carthage step and make sure to add `./Carthage -> ./Carthage/Cachefile`.
+2. set up a cache push step after your Carthage step and make sure to add:
+ * If you use Carthage v0.20.0 or later, `./Carthage`.
+ * Otherwise, `./Carthage -> ./Carthage/Cachefile`.
```

Comment:
Hi @chuganzy i think @trapacska tried to mention, that the cache step should update `./Carthage` directory if `./Cartfile.resolved` file changed.

Previously we tracked the `./Carthage/Cachefile` file and the cache updated the `./Carthage` directory if `Cachefile` changed.

So `The caching structure changed a little bit` means: the 'update indicator file' changed from `Cachefile` to `Cartfile.resolved`.

And as you mentiond from Carthage v0.21, the `Cartfile.resolved` tracks the swifts version.

So from Carthage v0.21 using `Cartfile.resolved` as  'update indicator file' equals our previous implementation (with `Cachefile`).